src/sentry/options/defaults.py

```diff
@@ -264,6 +264,15 @@
     flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK | FLAG_REQUIRED,
 )
 
+# API
+# Killswitch for apis to work with id or slug as path parameters
+register(
+    "api.id-or-slug-enabled",
+    default=False,
+    type=Bool,
+    flags=FLAG_MODIFIABLE_BOOL | FLAG_AUTOMATOR_MODIFIABLE,
```

Comment:
We want this flag to be modifiable modified live.

Reply:
Just had an offline conversation with Filippo, we do not need the `FLAG_PRIORITIZE_DISK` flag, `FLAG_AUTOMATOR_MODIFIABLE` is enough!